### PR TITLE
Fit encoding style between --save-json option and --save-xlsx option.

### PR DESCRIPTION
**Pull request by csh519**
The encoding style of `--svae-json` option and `--save-xlsx` option was different.
So there could be exist some different characters or encoding errors.

Changes:
- Add `_encode()` function.
- Use `json.encoder.py_encode_basestring_ascii()` for fit them.

### CODE REVIEW

File: cicd/missed_docstrings.txt [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-credsweeper/credentials/candidate.py:42:4: C0116: Missing function or method docstring (missing-function-docstring)
-credsweeper/credentials/candidate.py:50:4: C0116: Missing function or method docstring (missing-function-docstring)
-credsweeper/credentials/candidate.py:58:4: C0116: Missing function or method docstring (missing-function-docstring)
-credsweeper/credentials/candidate.py:66:4: C0116: Missing function or method docstring (missing-function-docstring)
-credsweeper/credentials/candidate.py:74:4: C0116: Missing function or method docstring (missing-function-docstring)
+credsweeper/credentials/candidate.py:13:0: C0115: Missing class docstring (missing-class-docstring)
+credsweeper/credentials/candidate.py:35:4: C0116: Missing function or method docstring (missing-function-docstring)
+credsweeper/credentials/candidate.py:43:4: C0116: Missing function or method docstring (missing-function-docstring)
+credsweeper/credentials/candidate.py:51:4: C0116: Missing function or method docstring (missing-function-docstring)
+credsweeper/credentials/candidate.py:59:4: C0116: Missing function or method docstring (missing-function-docstring)
```
Comment (babenek) [inline cicd/missed_docstrings.txt:8]: Setters-getters might be commented simple. It helps us reduce documentation loan.

**Review by babenek — COMMENTED** [review]
Please, merge my PR with unit test. 
Probably, xlsx file may be checked too.

File: cicd/missed_docstrings.txt (hunk 1 above)
Comment (csh519) [reply cicd/missed_docstrings.txt]: I added them by [13bfe52](https://github.com/Samsung/CredSweeper/pull/159/commits/13bfe522db92016b084ef8d261e88860f5ee34fe).
I'll make other PR for regarding docstrings.

I should to clean them all. :broom:

**Review by babenek — APPROVED** [review]
Looks good. 
Value "dkajc\u00f61" is looking similar in both reports.

**Review by kmnls — APPROVED** [review]

**Pull request merged**